diagnostic.md

```diff
@@ -32,22 +39,28 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+$('#div').on()
+$('li').on()
 
+$('#div').on(setter)
+$('li').on(setter)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is the place that translate document files from the server before it get displayed on the client side.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface to render web pages for you to view and interact with.